hermes-api/build.gradle

```diff
@@ -3,10 +3,11 @@ plugins {
 }
 
 dependencies {
-    compile group: 'org.hibernate', name: 'hibernate-validator', version: '5.0.0.Final'
+    compile group: 'org.hibernate.validator', name: 'hibernate-validator', version: '6.1.7.Final'
     compile group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.0'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: versions.jackson
     compile group: 'com.fasterxml.jackson.jaxrs', name: 'jackson-jaxrs-json-provider', version: versions.jackson
+    compile group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: versions.jackson
```

Comment:
@faderskd could you elaborate on that please? I see that we already use this dependency in `hermes-management`: https://github.com/allegro/hermes/blob/master/hermes-management/build.gradle#L27. Also, I tested changes from this PR in our environment and I have not seen any problems :).

Reply:
I meant that adding `jackson-datatype` would add some default serializers for `Instant` class and it could change the default json representation of the time (which is propably int timestamp as I remember). The tests probably don't notice it if they start using the same parser for converting json to DTO. I did not know that it is already used in management. I just wanted to emphasize that despite the serializing and parsing `Instant` change, the tests can still work but physical representation can change (which could cause problem in GUI).